src/feedback/trulens/feedback/dummy/endpoint.py

```diff
@@ -119,8 +120,80 @@ async def apost(
         payload: serial_utils.JSON,
         timeout: Optional[float] = None,
     ) -> Any:
-        # TODO: use async inside post
-        return self.post(url=url, payload=payload, timeout=timeout)
+        """Pretend to make an http post request to some model execution API."""
+
+        assert isinstance(payload, dict), "Payload should be a dict."
+
+        if timeout is None:
+            timeout = threading_utils.DEFAULT_NETWORK_TIMEOUT
+
+        # allocate some data to pretend we are doing hard work
+        temporary = np.empty(self.alloc, dtype=np.int8)  # noqa: F841
+
+        if self.delay > 0.0:
+            await asyncio.sleep(
+                max(0.0, self.ndt.np_random.normal(self.delay, self.delay / 2))
+            )
+
+        r = self.ndt.discrete_choice(
+            seq=["normal", "freeze", "error", "loading", "overloaded"],
```

Comment:
small nit - can we make these constants on the top?

Reply:
fixed